File: prime.java
public class prime 
{
    public static void main(String[] args)
    {
        int count=0,i=1,c=0;
        while (count<=20)
        {
            for(int j=1;j<=i;j++)
            {
                if(i%j==0)
                {
                    c++;
                }
            }
            if(c==2)
            {
                count++;
            }
            c=0;
            System.out.println(i);
            i++;
        }
    }
    
}

File: prime.c
#include <stdio.h>
int main()
{
    int n;
    int count = 0;
    scanf("%d", &n);
    for(int i = 0; i <= n; i++)
    {
        if(n%i==0)
        {
            count++;
        }
    }
    if(count==2)
        printf("prime");
    else
        printf("not prime");
}